lib/smack/SmackRep.cpp

```diff
@@ -1169,6 +1169,37 @@ void SmackRep::addBplGlobal(std::string name) {
   bplGlobals.push_back(name);
 }
 
+// Dealing with bitcasts between integers and floating-points
+// by adding appropriate assume statements constraining cast inverses
+const Stmt* SmackRep::inverseFPCastAssume(const Value* src, const Type* destType) {
+  if (!(SmackOptions::BitPrecise && SmackOptions::FloatEnabled)) {
+    return nullptr;
+  }
+  const Type *srcType = src->getType();
+  if (!(srcType->isFloatingPointTy() && destType->isIntegerTy())) {
+    return nullptr;
+  }
+  string fn = Naming::INSTRUCTION_TABLE.at(Instruction::BitCast);
```

Comment:
Very minor detail, but might be good to write `std::string` here instead of just `string`.

Reply:
Why? I am curious.

Reply:
It doesn't really matter, I just noticed we use `std::string` throughout the rest of the file.

Reply:
Good catch! It actually kind of matters, since we try to follow LLVM coding style: https://llvm.org/docs/CodingStandards.html#do-not-use-using-namespace-std